en/views/helpers/form.rst

```diff
@@ -1512,6 +1512,17 @@ example::
         'type' => 'radio'
     ]);
 
+Similar to input containers, the ``input()`` method will also attempt to use
+distinct templates for each form group. Form group is combo of label and input.
```

Comment:
"**A** Form group is **a** combo of label and input."
or 
"Form group**s** **are** combo**s** of label and input."